frontend/packages/ux-editor/src/components/config/componentSpecificContent/FileUpload/FileUploadComponent.tsx

```diff
@@ -105,7 +105,11 @@ export const FileUploadComponent = ({
           value={fileUploaderComponent.displayMode}
           propertyPath={`${component.propertyPath}/properties/displayMode`}
           renderField={({ fieldProps }) => (
-            <Radio.Group name={`${component.id}-display-mode`} inline={true} onChange={fieldProps.onChange}>
```

Comment:
For some reason, this file is formatted on commit. I've tried to roll it back, but it is formatted back again on commit.